core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1162,17 +1190,20 @@ class DAGScheduler(
         }
 
       case commitDenied: TaskCommitDenied =>
-        // Do nothing here, left up to the TaskScheduler to decide how to handle denied commits
+        // Do nothing here, left up to the TaskScheduler implementation
+        // to decide how to handle denied commits
 
       case ExceptionFailure(className, description, stackTrace, fullStackTrace, metrics) =>
-        // Do nothing here, left up to the TaskScheduler to decide how to handle user failures
+        // Do nothing here, left up to the TaskScheduler implementation
+        // to decide how to handle user failures
```

Comment:
Oh right, because of the extra word. That's fine.
PS You'll need to rebase this branch on master